### DISCUSSION TITLE
All image attributes in drag and drop

### DISCUSSION THREAD

**Discussion by ch2856** [joomla/joomla-cms#47083] [2. Cold Features]
### Is your feature request related to a problem? Please describe.

Drag and drop image to the article editor allow to add just the Alt tag

### Describe the solution you'd like
Allow to full option - Alt, Title. Lazy, Dimensions, caption. 
Cation should be a textbox and not a checkbox.

**Comment by charvimehradu** [comment]
I am unable to understand the solution described.Can you give more details about it? @ch2856 @chmst Thankyou in advance.

**Comment by ReLater** [comment]
> Can you give more details about it?

- Open Joomla 4 backend.
- Open an article or create a new one. Use TinyMce editor,
- Drag and drop an image from a local folder of your machine into the editor.
- You'll see

![grafik](https://user-images.githubusercontent.com/20780646/212991066-49579fe3-5244-4817-aed5-3838b31ca94a.png)

I entered "blubber".

Toggle editor to see HTML:

`<p><img src="images/800px_pinnacles_western_australia.jpg" alt="blubber" width="500" height="374" loading="lazy" /></p>`

I see lazy `loading` and I see `width` and `height,` reflecting the aspect ratio of the image, which is very important for lazy loading and shouldn't be changed from my point of view. Nowadys one should not use these attributes for sizing images. Only if the aspect ratio is the same after sizing. Maybe I'm wrong. My philosophy...

I don't see a need for title attribute from my point of view.

Caption should be a `figcaption` tag inside of of a `figure` tag.

But I don't know if that's wanted or possible in this edit context.

**Comment by ReLater** [comment]
When you have inserted the image and select it you can change all the wanted settings by clicking this button:

![grafik](https://user-images.githubusercontent.com/20780646/212993762-bf2edb9a-e49c-4cd1-ba31-8bbea2ddd8d3.png)

> Caption should be a textbox and not a checkbox.

The checkbox adds a placeholder in a `figcaption` tag in the editor field for entering a caption.

I don't see any advantages in asking for everything in the first pop-up. It's only annoying on small screens.
